src/main/java/step2/domain/Lotto.java

```diff
@@ -0,0 +1,44 @@
+package step2.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/*
+ * Lotto
+ * ver. 1.0
+ * 2020.05.31
+ * Copyright ...
+ */
+public class Lotto {
+
+    private List<Integer> lottoNumbers = new ArrayList<>();
+
+    private Lotto() {
+
+    }
+
+    public static Lotto of(int lottoDrawLimit, List<Integer> lottoGameNumbers) {
+
+        Lotto lotto = new Lotto();
+
+        // shuffle
+        Collections.shuffle(lottoGameNumbers);
+
+        // pick LOTTO_DRAW_LIMIT
+        lotto.lottoNumbers = drawLottoNumbers(lottoDrawLimit, lottoGameNumbers);
+        lotto.lottoNumbers.sort(Integer::compareTo);
+
+        return lotto;
```

Comment:
```suggestion
        List<Integer> lottoNumbers = drawLottoNumbers(lottoDrawLimit, lottoGameNumbers);
        lottoNumbers.sort(Integer::compareTo);
        return new Lotto(lottoNumbers);
```
생성자가 많아질수록 객체를 유연하게 사용할 수 있습니다. 
다양한 생성자를 만들어 보세요 !

Reply:
인자값이 다른 필요한 상황에 맞춰서 생성메소드를 만들어봤습니다.